draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -644,18 +592,50 @@ Whether to require Key Binding is up to the Verifier's policy,
 based on the set of trust requirements such as trust frameworks it belongs to.
 
 Other ways of proving Key Binding MAY be used when supported by the Verifier,
-e.g., when the Combined Format for Presentation without a Key Binding JWT is itself embedded in a
+e.g., when the presented SD-JWT without a Key Binding JWT is itself embedded in a
 signed JWT. See (#enveloping) for details.
 
-If no Key Binding JWT is included, the Combined Format for Presentation ends with
-the `~` character after the last Disclosure.
+## SD-JWT Structure {#sd-jwt-structure}
+
+An SD-JWT is composed of the following:
+
+* the Issuer-signed JWT
+* each respective Disclosure
+* optionally a Key Binding JWT
+
+The serialized format for the SD-JWT is the concatenation of each part delineated with a single tilde ('~') character as follows:
+
+```
+<JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<optional KB-JWT>
+```
+
+The order of the tilde separated values MUST be the Issuer-signed JWT, followed by any Disclosures, and lastly the optional Key Binding JWT.
```

Comment:
```suggestion
The values MUST be separated by the tilde in this order:
1. The Issuer-signed JWT
2. The Disclosures
3. The optional Key Binding JWT

```

Reply:
I prefer the text to the bullets/numbers here. Basically the same bullets are already above.

Reply:
ok